File: API/Shipments/Models/VehicleCorporateShipment.cs
namespace Transflower.EAgroServices.Shipments.Models;

public class VehicleCorporateShipment
{
    public int ShipmentId { get; set; }
    public int VehicleId { get; set; }
    public int CorporateId { get; set; }
    public string? VehicleType { get; set; }
    public string? RtoNumber { get; set; }
    public int Kilometers { get; set; }
    public string? Status { get; set; }
    public DateTime ShipmentDate { get; set; }
}

File: UI/LatestNGApp/UIAgroStdApp/src/app/Models/vehicle-corporate-shipment.ts
export class VehicleCorporateShipment {
    constructor(
        public shipmentId: number,
        public vehicleId: number,
        public corporateId: number,
        public vehicleType: string,
        public rtoNumber: string,
        public kilometers: string,
        public status: string,
        public shipmentDate: string,
        public companyName: string
    ) { }
}
